chains/orchestrator-relays/runtime/dancelight/src/tests/ethereum_token_transfers.rs

```diff
@@ -1338,6 +1341,267 @@ fn test_pricing_parameters() {
+                parents: 1,
+                interior: X2([
+                    GlobalConsensus(EthereumNetwork::get()),
+                    AccountKey20 {
+                        network: Some(EthereumNetwork::get()),
+                        key: token_address.into(),
+                    },
+                ]
+                .into()),
+            };
+
+            let asset_id = 42u16;
+
+            assert_ok!(ForeignAssetsCreator::create_foreign_asset(
+                root_origin(),
+                erc20_asset_location.clone(), // Use the ERC20 location
+                asset_id,
+                AccountId::from(ALICE),
+                true,
+                1
+            ));
+
+            // Give tokens to user as if tokens were bridged before + extra to check the correct
+            // amount is sent
+            ForeignAssets::mint_into(asset_id, &AccountId::from(BOB), amount_to_transfer + 10)
+                .expect("to mint amount");
+
+            // User tries to send tokens
+            let beneficiary_address = H160(hex!("0123456789abcdef0123456789abcdef01234567"));
+
+            // Beneficiary location must be represented using destination's point of view.
+            let beneficiary_location = Location {
+                parents: 0,
+                interior: X1([AccountKey20 {
+                    network: Some(EthereumNetwork::get()),
+                    key: beneficiary_address.into(),
+                }]
+                .into()),
+            };
+
+            let eth_asset = AssetId(erc20_asset_location.clone())
+                .into_asset(Fungibility::Fungible(amount_to_transfer));
+
+            let balance_before = Balances::balance(&AccountId::from(BOB));
+
+            assert_ok!(XcmPallet::transfer_assets(
+                RuntimeOrigin::signed(AccountId::from(BOB)),
+                Box::new(EthereumLocation::get().into()),
+                Box::new(beneficiary_location.into()),
+                Box::new(vec![eth_asset].into()),
+                0u32,
+                Unlimited,
+            ));
+
+            // Correct amount has been sent
+            assert_eq!(ForeignAssets::balance(asset_id, &AccountId::from(BOB)), 10);
+
+            // Check some fees have been payed
+            let balance_after = Balances::balance(&AccountId::from(BOB));
+            assert!(balance_before - balance_after > 0);
+        })
+}
+
+#[test]
+fn cant_send_eth_unknown_token() {
+    ExtBuilder::default()
+        .with_balances(vec![
+            (EthereumSovereignAccount::get(), 100_000 * UNIT),
+            (SnowbridgeFeesAccount::get(), 100_000 * UNIT),
+            (AccountId::from(ALICE), 100_000 * UNIT),
+            (AccountId::from(BOB), 100_000 * UNIT),
+        ])
+        .build()
+        .execute_with(|| {
+            // Setup bridge and token
+            let channel_id: ChannelId = ChannelId::new(hex!(
+                "00000000000000000000006e61746976655f746f6b656e5f7472616e73666572"
```

Comment:
where does this channel-id come from? is it random?

Reply:
Copied this value from other tests already present. Seems to come from a PR of yours @dimartiro , can you give us some insight?